testsuite/features/init_clients/proxy_container_branch_network.feature

```diff
@@ -25,6 +25,10 @@ Feature: Prepare the containerized branch server for PXE booting
     Then the "dhcp_dns" host should be present on private network
     And name resolution should work on private network
 
+  Scenario: Show the overview page of the containerized proxy
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
```

Comment:
I would read more natural if these two steps are part of the scenario below this one.

Reply:
OK. I just copied from traditional proxy where it is also like this.

Reply:
Suggestion applied